### DISCUSSION TITLE
db2/400 how to use SELECT command in files that have several members?

### DISCUSSION THREAD

**Discussion by lucianoww** [dbeaver/dbeaver#9412] [Q&A] [answered]
In Dbeaver 7.1.3 in connection db2 / 400 how is the SELECT command in files that have several members?

Example file FI739U, member B033003091

![screen1](https://user-images.githubusercontent.com/20689683/88933979-7595e680-d256-11ea-89aa-a6a4cbe9ea78.png)
![screen2](https://user-images.githubusercontent.com/20689683/88934806-71b69400-d257-11ea-9985-590d395ae574.png)

**Comment by kseniiaguzeeva** [comment]
Could you please clarify what is your problem? If you need to execute Select query, use:
 _Select * from FI739U_

**Comment by JMarkMurphy** [comment, marked as answer]
SQL does not support multi member files, directly. That is a DB2 for IBM i database construct. However, there are two solutions. You can spend $$$$$ and get DB2 Multi-system. That treats members as partitions, and then allows you to access those members. Or, you can use an alias to access a single member of a multi-member file. 

`CREATE OR REPLACE ALIAS alias name FOR table name (member name);`

The documentation for this statement is found here: [IBM i Knowledge Center](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_74/db2/rbafzhcalias.htm)

Alas, this is not a dBeaver issue.

**Comment by kseniiaguzeeva** [comment]
Unfortunately it cannot be resolved from our side.
